.goreleaser.yml

```diff
@@ -116,3 +116,29 @@ scoop:
     The New Relic CLI is an officially supported command line interface for New
     Relic, released as part of the Developer Toolkit.
   license: Apache-2.0
+
+nfpms:
+  - id: newrelic
+    package_name: newrelic-cli
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
+    builds:
+      - newrelic
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+    vendor: New Relic
+    homepage: https://github.com/newrelic/newrelic-cli
+    maintainer: opensource@newrelic.com
+    description: |
+      The New Relic CLI is an officially supported command line interface for New
+      Relic, released as part of the Developer Toolkit.
+    license: Apache 2.0
+    formats:
+      - deb
+      - rpm
+    bindir: /usr/bin
```

Comment:
Do we need to override this from `/usr/local/bin`? Or is this for consistency with the snapcraft / others?

Reply:
I think snapcraft is all `/usr/local/bin`, so I can make it match here.